### PR TITLE
Adding an Easyconfig for fastPHASE v1.1

### PR DESCRIPTION
**Pull request by rtripath89**

### CODE REVIEW

File: easybuild/easyconfigs/f/fastPHASE/fastPHASE-1.4.8.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+homepage = 'http://stephenslab.uchicago.edu/software.html#fastphase'
+description = """ fastPHASE: software for haplotype reconstruction, 
+and estimating missing genotypes from population data
+Documentation: http://scheet.org/code/fastphase_doc_1.4.pdf"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://scheet.org/code/']
+sources = ['Linuxfp.tar.gz']
```
Comment (boegel) [inline easybuild/easyconfigs/f/fastPHASE/fastPHASE-1.4.8.eb]: if there's no version included in the source tarball's name, please include a checksum as well

File: easybuild/easyconfigs/f/fastPHASE/fastPHASE-1.4.8.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+modextrapaths = {'PATH': ''}
```
Comment (boegel) [inline easybuild/easyconfigs/f/fastPHASE/fastPHASE-1.4.8.eb]: @rtripath89 this looks wrong, why is this needed if the binary is in a `bin` subdirectory (which is added to `$PATH` automagically by EB)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3c5ee871d28b19b95003e3880945a19c for a full test report.

File: easybuild/easyconfigs/f/fastPHASE/fastPHASE-1.4.8.eb (hunk 2 above)
Comment (rtripath89) [inline easybuild/easyconfigs/f/fastPHASE/fastPHASE-1.4.8.eb]: @boegel You are right. I am not sure why I added this line

File: easybuild/easyconfigs/f/fastPHASE/fastPHASE-1.4.8.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/f/fastPHASE/fastPHASE-1.4.8.eb]: @rtripath89 please include a checksum too?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6d756edfb007c2d8b610bb83335189fc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2408.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e9a03f4fd2c9cac0253f94800d115bbf for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @rtripath89!
